optuna/samplers/_tpe/sampler.py

```diff
@@ -74,11 +74,17 @@ class TPESampler(BaseSampler):
       <https://papers.nips.cc/paper/4443-algorithms-for-hyper-parameter-optimization.pdf>`_
     - `Making a Science of Model Search: Hyperparameter Optimization in Hundreds of
       Dimensions for Vision Architectures <http://proceedings.mlr.press/v28/bergstra13.pdf>`_
-    - `Multiobjective tree-structured parzen estimator for computationally expensive optimization
-      problems <https://dl.acm.org/doi/10.1145/3377930.3389817>`_
+    - `Tree-Structured Parzen Estimator: Understanding Its Algorithm Components and Their Roles for
+      Better Empirical Performance <https://arxiv.org/abs/2304.11127>`_
```

Comment:
I wonder where in the codebase this paper is from.

Reply:
I can also discuss whether the reference should include this paper (, which did not exist before) or not, but it actually makes sense to add this paper as long as the documentation says "For further information about TPE algorithm, please refer to the following papers:".

Just as a comment, I add the short summary of the paper.
The paper describes the control parameters of TPE and their effects on search in detail.
Importantly, each control parameter refers to the exact Optuna argument names in the paper.
The other four papers do not discuss each control parameter and its meaning.

Reply:
Thanks for the explanation. I see your point but it sound better to split the PR. 
Could you create another PR for adding the reference?